lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -901,6 +901,21 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => "Controls auto-instrumentation of resque at start up.  May be one of [auto|prepend|chain|disabled]."
         },
+        :'instrumentation.thread' => {
```

Comment:
@tannalynn, I vaguely remember you creating a new attribute for the DefaultSource that would set the default value in the config docs to be the string value. Is that in this branch? Should it be used here?

Reply:
Since the default value here is already string, it's not really needed since that is mostly to help default values that are procs that can't be evaluated when we generate the docs

Reply:
Riiiiight. Thanks for clarifying!